spec/controllers/hq/orphans_controller_spec.rb

```diff
@@ -2,16 +2,47 @@
 require 'will_paginate/array'
 
 RSpec.describe Hq::OrphansController, type: :controller do
-  let(:orphan) {build_stubbed :orphan}
```

Comment:
@gmunumel My comment about using `@` instead of `let` only applied to view specs. Elsewhere, `let` is preferred. Don't worry about changing it back here, it's not a huge deal.